src/pages/understand-stacks/running-testnet-node.md

```diff
@@ -93,6 +93,14 @@ Clone this repository:
 git clone https://github.com/blockstack/stacks-blockchain.git; cd stacks-blockchain
 ```
 
+Optionally, set env vars for a specific version, commit hash and branch:
+
+```bash
+export STACKS_NODE_VERSION=2.0.9
```

Comment:
Can we add a comment here saying something like "Change the below values to reflect the version, branch, and git commit of the source code being built for accuracy. The following values are just an example."